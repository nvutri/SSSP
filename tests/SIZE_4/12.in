c random graph n=32 m=128 lo=0 hi=16777216 seed=356728362
p sp 32 128
a 17 26 10488305
a 26 5 11718222
a 5 3 6463774
a 3 4 6925189
a 4 24 16542265
a 24 20 90735
a 20 16 995824
a 16 25 2688190
a 25 31 6621578
a 31 6 7473529
a 6 32 11030654
a 32 14 2208112
a 14 28 2117760
a 28 29 2795555
a 29 15 5100379
a 15 30 11524223
a 30 1 3603450
a 1 8 12716772
a 8 2 6511931
a 2 22 16258569
a 22 18 12765089
a 18 12 6776370
a 12 19 13542727
a 19 21 13532606
a 21 10 11513536
a 10 13 7854006
a 13 7 5991575
a 7 9 13669814
a 9 27 7704723
a 27 23 14131755
a 23 11 9098329
a 11 17 8243310
a 29 4 4524796
a 22 28 5570327
a 10 3 14606990
a 7 21 225771
a 8 6 14647627
a 10 9 1398302
a 24 12 5197834
a 3 17 10795475
a 15 2 3168388
a 17 10 847926
a 22 6 2107084
a 29 30 4429138
a 28 22 993074
a 25 6 15938808
a 19 16 6066336
a 9 12 15488010
a 19 14 1671233
a 28 25 7750717
a 24 12 14206138
a 1 2 9261994
a 14 26 7356814
a 13 20 13500549
a 29 7 15229714
a 8 1 4522423
a 10 27 4167374
a 28 22 16156288
a 15 6 3715833
a 3 12 3978808
a 19 16 14522744
a 15 31 9148999
a 31 24 6538863
a 29 4 16501902
a 7 3 5767883
a 16 7 1795075
a 19 32 2407789
a 22 14 8314818
a 20 2 14240013
a 26 2 7074316
a 5 2 8310483
a 12 8 11407914
a 8 20 7199725
a 7 32 4809995
a 20 11 15878009
a 13 10 2404966
a 30 23 15108897
a 23 1 1457215
a 18 4 3891220
a 12 29 11840997
a 22 5 3566161
a 15 14 12428703
a 1 4 8896896
a 27 10 14051861
a 18 12 5647570
a 5 10 13236868
a 10 20 2556420
a 25 15 7151677
a 22 23 11679963
a 6 21 4883796
a 1 28 2858955
a 12 29 14834704
a 5 32 5936775
a 4 5 3952755
a 21 20 10340585
a 18 24 12581125
a 3 32 9896700
a 1 15 8323246
a 29 1 14487942
a 32 14 1473675
a 26 6 8201201
a 31 3 8119169
a 31 29 7816295
a 13 5 2281408
a 17 18 5118238
a 5 23 4694707
a 15 7 13421201
a 24 8 4582112
a 1 10 4272334
a 25 18 9845689
a 21 10 3588328
a 25 24 2064323
a 28 17 4225056
a 16 32 5642993
a 7 9 4200473
a 15 6 5872121
a 4 32 6917792
a 12 5 9416081
a 12 24 16127613
a 23 18 9703030
a 6 5 3488843
a 17 31 3167022
a 5 15 9072070
a 27 29 15444087
a 32 21 13529774
a 1 19 1313258
a 15 1 16621596
a 25 14 14310270
