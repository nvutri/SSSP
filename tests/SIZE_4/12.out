1 0
2 9.26199e+06
3 1.3882e+07
4 8.8969e+06
5 7.41819e+06
6 5.95911e+06
7 9.17467e+06
8 1.27168e+07
9 5.67064e+06
10 4.27233e+06
11 2.12112e+07
12 1.73299e+07
13 1.21263e+07
14 2.98449e+06
15 8.32325e+06
16 7.37959e+06
17 7.08401e+06
18 1.22022e+07
19 1.31326e+06
20 6.82875e+06
21 9.40044e+06
22 3.85203e+06
23 1.21129e+07
24 1.21321e+07
25 1.00678e+07
26 1.03413e+07
27 8.43971e+06
28 2.85896e+06
29 5.65451e+06
30 1.00836e+07
31 1.0251e+07
32 3.72105e+06
